docker/apps/kona_app_generic.dockerfile

```diff
@@ -8,7 +8,7 @@ FROM ubuntu:22.04 AS dep-setup-stage
 SHELL ["/bin/bash", "-c"]
 
 # Install deps
-RUN apt-get update && apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing \
```

Comment:
There's a PR up for this @ https://github.com/op-rs/kona/pull/1890 - can we keep this separate? re: the comment there about this skipping dependencies that can't be installed.